src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_DIMENSION = 10;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int figureType = random.nextInt(5);

```

Comment:
The number '5' is a magic number. You should replace it with a named constant to make it clear what it represents. According to the checklist, all magic numbers should be constants.